libraries/core_libs/network/src/tarcap/packets_handlers/dag_sync_packet_handler.cpp

```diff
@@ -129,7 +143,7 @@ void DagSyncPacketHandler::process(const PacketData& packet_data, const std::sha
       std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch()).count();
   peer->peer_dag_syncing_ = false;
 
-  LOG(log_dg_) << "Received DagSyncPacket with blocks: " << received_dag_blocks_str
+  LOG(log_er_) << "Received DagSyncPacket with blocks: " << dag_blocks_to_log
```

Comment:
why error ?

Reply:
Reverted to debug. It was changed to error for some debugging that I did and I forget to change it back.